crates/evm/evm/src/executors/invariant/mod.rs

```diff
@@ -29,7 +29,7 @@ use proptest::{
     test_runner::{TestCaseError, TestRunner},
 };
 use revm::{
-    primitives::{Address as rAddress, HashMap, U256 as rU256},
+    primitives::{Address as aAddress, HashMap, U256 as rU256},
```

Comment:
I think we can just use `alloy_primitives::Address` now

Reply:
Anvil stuff, I don't want to get into that